lib/omnibus-software/version.rb

```diff
@@ -0,0 +1,3 @@
+module OmnibusSoftware
+  VERSION = "5.0.0"
```

Comment:
let's use `4.0.0` as it comes from upstream, not sure where it's used but not changing it could be ease further pulls from them